src/components/App.test.tsx

```diff
@@ -1,10 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { Route } from 'react-router-dom';
-import App from './App';
-
-it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Route component={App} />, div);
-    ReactDOM.unmountComponentAtNode(div);
```

Comment:
Why was this test removed out? What was the error?

Reply:
The error was this: Invariant Violation: You should not use <Route> or withRouter() outside a <Router>. 
It's due to our app using react-router and has dependencies to that. It would be possible to test it if we mock the router.

Reply:
Added this test back now